tools/dashboard-plugin-template/src/plugin/Plugin.tsx

```diff
@@ -10,7 +10,8 @@ import {
     newCustomWidget,
 } from "@gooddata/sdk-ui-dashboard";
 
-import packageJson from "../../package.json";
+import entryPoint from "../plugin_entry";
```

Comment:
Will this work with the plugin toolkit? Will the plugin_ be replaced here?

Reply:
Oh, good catch! `tools/plugin-toolkit/src/initCmd/index.ts` updated.